### PR TITLE
APPS-632 Adding currency formatting based on provided metadata

### PR DESCRIPTION
**Pull request by jkotrs**
https://jira.internal.dnanexus.com/browse/APPS-632

### CODE REVIEW

**Review by mhrvol — COMMENTED** [review]
LGTM, but update the changelog - add change description to unreleased - an example of this can be seen here:
https://github.com/dnanexus/dx-toolkit/pull/679/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Comment by jkotrs** [comment]
Thanks Marek, will do

**Review by mhrvol — APPROVED** [review]

File: src/python/dxpy/utils/describe.py [hunk 1]
```diff
@@ -918,6 +918,14 @@ def print_execution_desc(desc):
         if field not in recognized_fields:
             print_json_field(field, desc[field])
 
+def print_currency(value, meta):
+    prefix = '-' if value < 0 else ''
+    prefix += meta['symbol'] if meta['symbolPosition'] == 'left' else ''
+    suffix = f' {meta["symbol"]}' if meta['symbolPosition'] == 'right' else ''
```
Comment (kpjensen) [inline src/python/dxpy/utils/describe.py:924]: Are f strings supported in Python 2.7? If not, we probably need to change to `str.format()` here for compatibility.  

We have not yet officially deprecated Python 2.7 support, still building for the 16.04 job execution environment and tarball published to the Downloads page in our docs.

**Review by kpjensen — CHANGES_REQUESTED** [review]

File: src/python/dxpy/utils/describe.py (hunk 1 above)
Comment (jkotrs) [reply src/python/dxpy/utils/describe.py:924]: I see, I assumed we've moved over to python 3+ based on the commend in README.md (I think it states April 2021 as end of support). I can rewrite to 2.7 np

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
Pending changes from Kurt's comment

**Review by r-i-v-a — APPROVED** [review]

**Review by kpjensen — APPROVED** [review]

**Pull request merged**